src/helper-modules/govuk-notify/govuk-notify.service.ts

```diff
@@ -0,0 +1,45 @@
+import { BadRequestException, ForbiddenException, Injectable, UnauthorizedException, UnprocessableEntityException } from '@nestjs/common';
+import { NotifyClient } from 'notifications-node-client';
+
+import { PostEmailsRequestItemDto } from '../../modules/emails/dto/post-emails-request.dto';
+import { PostEmailsResponseDto } from './dto/post-emails-response.dto';
+
+@Injectable()
+export class GovukNotifyService {
+  constructor() {}
+
+  async sendEmail(govUkNotifyKey: string, email: PostEmailsRequestItemDto): Promise<PostEmailsResponseDto> {
```

Comment:
We don't use JSDoc's in MDM, and current typescript hinting/IntelliSense is quite good.

Reply:
Added Exceptions to function signatures, but this is quite noisy and we didn't use this practice before!, I could revert this change if no exceptions option is prefered.

Reply:
I added exceptions to return value on Friday, but still not sure if it is required.

Other languages use `throws` keyword to describe functions exceptions. It is because exceptions are not returned, they thrown, and it is not same as return types. Looks like typescript is missing exception documentation.

https://www.learningtypescript.com/articles/why-typescript-doesnt-include-a-throws-keyword

This is commit for JSDoc, but I not sure if we need it - https://github.com/UK-Export-Finance/mdm-api/pull/820/commits/c8b6de03ee82b4e6f5d2b30e010eb1ef4bdf67cc